src/main/java/de/qabel/desktop/ui/sync/item/SyncItemController.java

```diff
@@ -144,17 +144,19 @@ public void delete() {
                             if (buttonType != ButtonType.YES) {
                                 return;
                             }
-                            if (syncConfig.getSyncer() != null) {
-                                try {
-                                    syncConfig.getSyncer().stop();
-                                } catch (InterruptedException e) {
-                                    // best effort
-                                    alert("error while stopping sync: " + e.getMessage(), e);
-                                } finally {
-                                    syncConfig.setSyncer(null);
-                                    boxSyncRepository.delete(syncConfig);
+                            boxSyncRepository.delete(syncConfig);
+                            new Thread(() -> {
+                                if (syncConfig.getSyncer() != null) {
+                                    try {
+                                        syncConfig.getSyncer().stop();
+                                    } catch (InterruptedException e) {
+                                        // best effort
+                                        alert("error while stopping sync: " + e.getMessage(), e);
+                                    } finally {
+                                        syncConfig.setSyncer(null);
```

Comment:
You spawn a new thread and then stop a syncer there? Why is there a syncer in a new thread?